binding.gyp

```diff
@@ -40,23 +39,18 @@
             '-ltensorflow',
           ],
           'library_dirs' : ['<(PRODUCT_DIR)'],
-          'variables': {
-            'tensorflow-library-target': 'linux-cpu',
-          },
           'actions': [
             {
-              'action_name': 'get_libtensorflow',
+              'action_name': 'symlink',
```

Comment:
With the name change - this becomes something like `'action_name': 'deps-stage'`

Reply:
Done.